app/Http/Middleware/VerifyCsrfToken.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace App\Http\Middleware;
+
+use Illuminate\Foundation\Http\Middleware\VerifyCsrfToken as Middleware;
+
+class VerifyCsrfToken extends Middleware
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Missing doc comment for class VerifyCsrfToken](https://app.codacy.com/gh/BuildForSDG/Team-059-Product/pullRequest?prid=5478046)